### PR TITLE
updated grating enumeration

### PR DESCRIPTION
**Pull request by parfa30**
This should be merged as part of xml-23. Also, when merged, will need to change some of the scripts so that they align with the updated configuration.

### CODE REVIEW

File: doc/news/DM-45475.misc.rst
Comment (couger01) [inline doc/news/DM-45475.misc.rst:1]: I would remove the part about Windows 10 and simply leave it as updating the grating enumeration.

**Review by couger01 — CHANGES_REQUESTED** [review]
One requested change.

**Review by couger01 — APPROVED** [review]
LGTM.

**Review by couger01 — CHANGES_REQUESTED** [review]
Change the news fragment to use Electrometer section not misc.

**Comment by parfa30** [comment]
I don't know what you mean. Do you mean change the name of the news fragment to be DM-45475.Electrometer.rst? This is for the monochromator.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**